### PR TITLE
add current and expected jsons with transformer for edx.course.enrollment.upgrade.succeeded

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/2HRvOHFn/69-enrollment-event-edxcourseenrollmentupgradesucceeded_

**Description:** _The server emits this event when the process of upgrading a student’s student_courseenrollment.mode from ‘audit’ or ‘honor’ to ‘verified’ is complete._

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.succeeded.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "ip": "116.58.62.58",
+            "page": null,
+            "path": "/shoppingcart/postpay_callback/",
+            "session": "f3029807d5ac0b71b59b53e9e789f0fc"
+        }
+    },
+    "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
+    "object": {
+        "id": "https://tasawernawaz.sandbox.edx.org/shoppingcart/payment_fake/",
+        "type": "Entity",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.succeeded.json]: This is a very generic type. Is there no specific Entity type for this event?

Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.succeeded.json]: i have changed the event, kindly give me another review

File: openedx/features/caliper_tracking/transformers/enrollment_transformers.py [hunk 2]
```diff
@@ -145,3 +145,39 @@ def edx_course_enrollment_upgrade_clicked(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_course_enrollment_upgrade_succeeded(current_event, caliper_event):
+    """
+
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/enrollment_transformers.py]: Kindly add doc string here.

Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/enrollment_transformers.py]: Kindly add doc string here.

**Pull request merged**